Compilation of hierarchy-builder fails

It seems that trying to install hierarchy-builder (for Coq 8.20) with opam now fails after having successfully installed coq-elpi 3.0.0:

```
   -> installed elpi.3.0.1
  -> installed rocq-elpi.3.0.0
  -> installed coq-elpi.3.0.0
  Error:  The compilation of rocq-hierarchy-builder.1.9.1 failed at "make build".
  
  #=== ERROR while compiling rocq-hierarchy-builder.1.9.1 =======================#
  # context              2.3.0 | linux/x86_64 | ocaml-option-flambda.1 ocaml-variants.4.13.1+options | https://coq.inria.fr/opam/released#2025-07-22 07:30
  # path                 ~/.opam/4.13.1+flambda/.opam-switch/build/rocq-hierarchy-builder.1.9.1
  # command              /usr/bin/make build
  # exit-code            2
  # env-file             ~/.opam/log/rocq-hierarchy-builder-164-eccb39.env
  # output-file          ~/.opam/log/rocq-hierarchy-builder-164-eccb39.out
  ### output ###
  # [...]
  # COQDEP VFILES
  # COQC HB/structures.v
  # File "./HB/structures.v", line 299, characters 0-44:
  # Error:
  # Two types for the same symbol cannot only differ on modes or functionality.
  # Previous declaration: File "/home/coq/.opam/4.13.1+flambda/.opam-switch/build/rocq-elpi.3.0.0/elpi/coq-lib.elpi", line 133, column 0, characters 5272-5314:
  # 
  # make[2]: *** [Makefile.coq:818: HB/structures.vo] Error 1
  # make[2]: *** [HB/structures.vo] Deleting file 'HB/structures.glob'
  # make[1]: *** [Makefile.coq:417: all] Error 2
  # make[1]: Leaving directory '/home/coq/.opam/4.13.1+flambda/.opam-switch/build/rocq-hierarchy-builder.1.9.1'
  # make: *** [Makefile:120: this-build] Error 2
```

Could it be an insufficient version constraint?

--- a/released/packages/rocq-hierarchy-builder/rocq-hierarchy-builder.1.9.0/opam
+++ b/released/packages/rocq-hierarchy-builder/rocq-hierarchy-builder.1.9.0/opam
@@ -12,8 +12,8 @@ build: [ [ make "build"]
 install: [ make "install" ]
 depends: [
   ("coq" {>= "8.18" & < "8.20~"} & "coq-elpi" {>= "2.0"}
-  | "coq" {>= "8.20" & < "8.21~"} & "coq-elpi" {>= "2.4" | = "dev"}
-  | "rocq-core" {(>= "9.0" & < "9.1~") | = "dev"} & "rocq-elpi" {>= "2.4" | = "dev"})
+  | "coq" {>= "8.20" & < "8.21~"} & "coq-elpi" {>= "2.4" & < "3.0~"}
+  | "rocq-core" {>= "9.0" & < "9.1~"} & "rocq-elpi" {>= "2.4" & < "3.0~"})
 ]
 conflicts: [
   "coq-hierarchy-builder" {< "1.9~"}

--- a/released/packages/rocq-hierarchy-builder/rocq-hierarchy-builder.1.9.1/opam
+++ b/released/packages/rocq-hierarchy-builder/rocq-hierarchy-builder.1.9.1/opam
@@ -12,8 +12,8 @@ build: [ [ make "build"]
 install: [ make "install" ]
 depends: [
   ("coq" {>= "8.18" & < "8.20~"} & "coq-elpi" {>= "2.0"}
-  | "coq" {>= "8.20" & < "8.21~"} & "coq-elpi" {>= "2.4" | = "dev"}
-  | "rocq-core" {(>= "9.0" & < "9.1~") | = "dev"} & "rocq-elpi" {>= "2.4" | = "dev"})
+  | "coq" {>= "8.20" & < "8.21~"} & "coq-elpi" {>= "2.4" & < "3.0~"}
+  | "rocq-core" {>= "9.0" & < "9.1~"} & "rocq-elpi" {>= "2.4" & < "3.0~"})
 ]
 conflicts: [
   "coq-hierarchy-builder" {< "1.9~"}

